problems/8.types.js

```diff
@@ -17,10 +17,14 @@
  * @param {number} b - the second number
  * @returns {boolean} - if a and b are both number types, return true
  */
-function bothNumbers() {
-
+function bothNumbers(num1, num2) {
+  let a = true
+  let b = true
+  return a && b;
 }
```

Comment:
This would always return true. What you need to do is see if the arguments are both of type number.